### PR TITLE
Update Figgy to Ruby 2.6/Bundler 2.x

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: group_vars/figgy_staging.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ figgy_rabbit_user: '{{vault_figgy_staging_rabbit_user}}'
 figgy_rabbit_password: '{{vault_figgy_staging_rabbit_password}}'
 figgy_rabbit_host: 'figgy-staging1.princeton.edu'
 figgy_rabbit_server: 'amqp://{{figgy_rabbit_user}}:{{figgy_rabbit_password}}@{{figgy_rabbit_host}}:5672'
+figgy_repository_mount: '/mnt/diglibdata/hydra_binaries/figgy_staging'
```
Comment (hackmastera) [inline group_vars/figgy_staging.yml]: Does this have an issue? Doesn't seem related to ruby / bundler versions.

Comment (tpendragon) [reply group_vars/figgy_staging.yml]: It doesn't, I just couldn't run the playbook without it.

Comment (hackmastera) [reply group_vars/figgy_staging.yml]: Seems important. I wonder how it came to be missing. Could you split it into its own commit, at least?

Comment (tpendragon) [reply group_vars/figgy_staging.yml]: Sure.
